airgun/views/host.py

```diff
@@ -203,7 +203,7 @@ class HostsView(BaseLoggedInView, SearchableViewMixin):
         './/table',
         column_widgets={
             0: Checkbox(
-                locator=".//input[@class='host_select_boxes']"),
+                locator="//input[@id='check_all']"),
```

Comment:
I'm not sure about this change. Have you tried running `session.host.delete_hosts(['host1', 'host2'])`? Seems that instead of selecting these specific hosts, it will click `<input id="check_all">` multiple times.